bundles/org.openhab.binding.http/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -203,6 +235,10 @@
 	<channel-type id="switch">
 		<item-type>Switch</item-type>
 		<label>Switch Channel</label>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Mode</tag>
+		</tags>
```

Comment:
Too generic to be tagged in a meaningful way, or..?

Reply:
Or what?

Reply:
Removing the tags